rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/UpdateCommandQuery.java

```diff
@@ -74,4 +74,17 @@ protected Builder getThis() {
             return this;
         }
     }
+
+    private enum QueryParameter {
+
+        RECORD(1),
+        STATUS(2),
+        ID(3);
+
+        private final int index;
```

Comment:
Let's run the IDEA inspection on this.

I think the Codacy is confused by the absence of a getter.